stock_barcodes/models/stock_quant.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import models
+
+
+class StockQuant(models.Model):
+    _inherit = "stock.quant"
+
+    def action_barcode_inventory_quant_unlink(self):
+        self.with_context(inventory_mode=True).action_set_inventory_quantity_to_zero()
+
+    def _get_fields_to_edit(self):
+        return [
+            "location_id",
+            "product_id",
+            "product_uom_id",
+            "lot_id",
+            "package_id",
+            "product_qty",
+        ]
+
+    def action_barcode_inventory_quant_edit(self):
+        wiz_barcode_id = self.env.context.get("wiz_barcode_id", False)
+        wiz_barcode = self.env["wiz.stock.barcodes.read.inventory"].browse(
+            wiz_barcode_id
+        )
+        for quant in self:
+            for fname in quant._fields:
+                if fname in self._get_fields_to_edit():
+                    wiz_barcode[fname] = quant[fname]
+            wiz_barcode.product_qty = quant.inventory_quantity
```

Comment:
When reading this code, I see that, on line 30, in the loop on quant._fields that are in _get_fields_to_edit(), it will write:
```
wiz_barcode.product_qty
```
then, on line 31, it will write:
```
wiz_barcode.product_qty = quant.inventory_quantity
```
Is it normal ? Shouldn't we remove "product_qty" from _get_fields_to_edit ?